workflows_core/utils/keyphrase.py

```diff
@@ -0,0 +1,37 @@
+from typing import List
+
+
+class Keyphrase:
+    def __init__(self,
+                 _id: str,
+                 text: str,
+                 ancestors: List[str] = [],
+                 parents: List[str] = [],
+                 level: int = 0,
+                 keyphrase_score: float = 0.0,
+                 frequency: int = 0,
+                 metadata=None,
+                 ):
```

Comment:
Please add an example on how to use this as a docstring.  Do not use lists as default values as they are immutable objects. Use `None` as default.